charts/manifests/templates/utils/README.md

```diff
@@ -49,14 +52,86 @@ String
 {{- include "bedag-lib.utils.common.fullname" $ }}
 ```
 
+### SelectorLabels
+---
+
+SelectorLabels Wrapper. Adds Bundlename to SelectorLabels, if defined.
+
+#### Arguments
+
+If a required argument is missing, the template engine will intentionally fail.
+
+  * `.`/`.context` - Inherited Root Context (Required).
+  * `.bundlename` - Adds the bundlename (Optional)
+
+**Note**: Implements the `{{ lib.utils.common.selectorLabels }}` template and supports all it's arguments/keys.
+
+#### Returns
+
+String
+
+#### Usage
+
+```
+{{- include "bedag-lib.utils.common.commonLabels" $ }}
```

Comment:
wrong usage, correct is: `{{- include "bedag-lib.utils.common.selectorLabels" $ }}`